### ISSUE TITLE
Accessibility Issue: HTML document does not have lang attribute /presentation.html

### ISSUE THREAD

**Issue by Moro-Code** [sara-sabr/ITStrategy#733]
**Description**

Every HTML document must have a lang attribute. 

**Device Information**  
*Device*:  
*Operating System (incl. version)*:  
*Browser (incl. version)*:  
*Additional Details*:

**Element(s) Information**  
*Page*:  https://sara-sabr.github.io/ITStrategy/presentation.html
*CSS Selector(s) (if possible and applicable)*:  html
*Additional Identifier(s) (if possible and applicable)*:  
*Additional Details*:  

**Screenshot(s) (if possible)**

**Additional information or thoughts**

As a result of scan  #698
